app/README.md

```diff
@@ -37,3 +42,60 @@ npm run start:app
 ```
 
 You can open the Measure Repository Service frontend application by navigating to [http://localhost:3001](http://localhost:3001) in your browser.
+
+
+# Frontend Features
+
+## Navigation
+
+The application is divided into two main sets of functionality, _Repository_ and _Authoring_, both of which are accessible from tabs at the top of the page.
+
+<img src="./static/tabs.png" alt="Screenshot of tabs" width="200"/>
+
+The Repository tab provides functionality for access and interacting with measure artifacts that are in the FHIR Measure Repository Service. The Authoring tab focuses on measure artifacts that are in the process of being drafted. Artifacts can be copied or moved between these two spaces through _draft_ and _release_ actions, described in the [Viewing](#viewing) and [Editing](#editing) sections.
+
+Within these tabs, the left panel navigation allows for selecting artifact types or searching (Repository only) for artifacts so that they can be browsed for viewing or actions.
+
+<img src="./static/navigation.png" alt="Screenshot of navigation" width="250"/>
+
+## Landing Page
+The landing page can be accessed on first load or by clicking the Measure Repository home button on the top left. This page links to resources to learn more about the Measure Repository and shows existing FHIR service capabilities.
+
+## Repository Tab
+The Repository tab displays sets of artifacts from the FHIR Measure Repository Service. Options allow for viewing resource details or reviewing the resource.
+
+<img src="./static/repository-options.png" alt="Screenshot of repository options" width="75"/>
+
+### Viewing
+-	Measure: Resource details include options to see the JSON, Data Requirements, or Dependencies.        
+-	Library: Resource details include options to see the JSON or Narrative.  
+-	Drafting: Both resource views allow the user to create a draft from the artifact. This action copies the existing artifact and any children it owns to the draft Authoring space with an incremented version to logically differentiate it from the original artifact.
+
+### Reviewing
+Reviewing a resource provides options for viewing existing comments on the resource or adding a new comment. (Note: this is currently partially implemented and requires updates for viewing comments.)
+
+## Authoring Tab
+The Authoring tab gives options for creating a new draft artifact from scratch or starting from a copy of an existing artifact from the FHIR Measure Repository. 
+
+<img src="./static/draft-artifact.png" alt="Screenshot of create draft artifact" width="350"/>
+
+The left navigation resource type selection displays sets of artifacts that are in progress, being drafted for potential addition to the FHIR Measure Repository. Options allow for editing, reviewing, or deleting a draft resource.
```

Comment:
```suggestion
The left navigation resource type selection displays sets of artifacts that are in draft status. Options allow for editing, reviewing, or deleting a draft resource.
```
I am not totally sure what's best here I will leave it up to you but I think that sentence could be slightly reworded?

Reply:
The app really sits in a bit of a weird space - I wanted to make clear that the draft measures are not in the FHIR repository but rather just in our local storage until they are "released" (or otherwise moved to the official FHIR repo).

Reply:
Ah I see what you mean. I think you can keep essentially what you have but should there be an "and" instead of the comma? Idk grammar is hard

Reply:
agreed on the "and"!
```suggestion
The left navigation resource type selection displays sets of artifacts that are in progress and being drafted for potential addition to the FHIR Measure Repository. Options allow for editing, reviewing, or deleting a draft resource.
```